lib/xebow/rgb_matrix.ex

```diff
@@ -13,8 +13,8 @@ defmodule Xebow.RGBMatrix do
     defstruct @fields
   end
 
-  @type pixels :: list({non_neg_integer, non_neg_integer})
-  @type colors :: list(any)
+  @type pixel :: {non_neg_integer, non_neg_integer}
+  @type pixel_color :: Chameleon.HSV.t()
```

Comment:
I didn't want animations to be forced to use `Chameleon.HSV`, and Chameleon doesn't provide a type representing "any color", which is why I just left it as `any` temporarily, until I could either clean up Chameleon itself (which hasn't been touched in a while and could use some maintenance), or define a type for it myself.

Before actually painting the color to the spidev, we use `Chameleon.convert`, which handles any of Chameleon's supported color types.

Reply:
That makes sense. I'm ok with changing it back to `any`. I'd also be happy to add in an `any_color` type that we could use until Chameleon is updated if you'd like. Maybe something like this if it seems reasonable?

```ex
@type any_color ::
  Chameleon.Color.RGB.t()
  | Chameleon.Color.CMYK.t()
  | Chameleon.Color.Hex.t()
  | Chameleon.Color.HSL.t()
  | Chameleon.Color.HSV.t()
  | Chameleon.Color.Keyword.t()
  | Chameleon.Color.Pantone.t()
```

Reply:
Yes, exactly that; it's annoying that Chameleon doesn't provide that type.

Reply:
Slight update to the naming since I got nit picky: `any_color_model`. This is because "color" implies a concrete value like `red` or `255, 0, 0`, where as "color model" implies the system used to represent a range of colors. <sup>[ref](https://en.wikipedia.org/wiki/HSL_and_HSV)</sup>